src/devices/UFireIse/samples/UFireIse.Sample.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.UFire.Sample
+{
+    /// <summary>
+    /// This sample is for Raspberry Pi Model 3B+
+    /// </summary>
+    public static class Program
+    {
+        private const int BusId = 1;
+
+        private static void PrintHelp()
+        {
+            Console.WriteLine("Command:");
+            Console.WriteLine("    B           Basic");
+            Console.WriteLine("    O           Orp");
+            Console.WriteLine("    P           PH");
+            Console.WriteLine();
+        }
+
+        /// <summary>
+        /// Example program main entry point
+        /// </summary>
+        /// <param name="args">Command line arguments see <see cref="PrintHelp"/></param>
+        public static void Main(string[] args)
+        {
+            I2cConnectionSettings settings = new I2cConnectionSettings(BusId, UFireIse.I2cAddress);
+            I2cDevice device = I2cDevice.Create(settings);
```

Comment:
1. you should always wrap this in using statement
2. only open when you actually use it (i.e. not only printing help)